src/main/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/indexstatemanagement/model/managedindexmetadata/StepMetaData.kt

```diff
@@ -48,11 +48,12 @@ data class StepMetaData(
     }
 
     override fun toXContent(builder: XContentBuilder, params: ToXContent.Params): XContentBuilder {
-        return builder.startObject(STEP)
```

Comment:
Previously we don't show step metadata out as json. The change here is to be consistent with ActionMetadata [code link](https://github.com/opendistro-for-elasticsearch/index-management/blob/eb668010f58699747b37192d033402a1ec75ea8e/src/main/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/indexstatemanagement/model/managedindexmetadata/ActionMetaData.kt#L58-L64)